Userland/Libraries/LibC/string.h

```diff
@@ -11,6 +11,17 @@
 
 __BEGIN_DECLS
 
+// There are a few C Standard Libraries that have completely broken the standard
+// and include <strings.h> inside <string.h>. This means that certain pieces
+// of software expect functions in <strings.h> to be available by including
+// <string.h>...
+// Considering that we'd rather be "on-spec" rather than compatible, let's
+// hide this behind a macro that can be used per port instead of leaking the
+// prototypes into the namespace as part of <string.h>
+#ifdef __STRINGS_H_COMPAT_HACK
```

Comment:
Let me know if this is an inappropriate macro name. I wanted something that wasn't obvious that requires someone to type out `HACK` so that they know they're going to be polluting the namespace with a header they might not expect to be included.